From b7d25c1d77cc6a626e8f839f3a6f3479d05a49bd Mon Sep 17 00:00:00 2001
From: AnilKumar Ch <anilkumar@ti.com>
Date: Wed, 15 Aug 2012 10:14:58 +0800
Subject: [PATCH 508/609] arm: omap: am335x: Add TI i2c sensors data

Original commit: 62d2d5372149416d208d0ab4f2b6e72016077738
		 9c4f7865282b24df57360c885cbdfdc2fd0168cf
  From git://arago-project.org/git/projects/linux-am33x.git
  And ti-sdk-am335x-evm-05.05.00.00-Linux-x86-Install image

Add ambient light sensor and TI TMP275 temperature sensor support

Signed-off-by: AnilKumar Ch <anilkumar@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index b46328c..cc7a9fb 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -1528,6 +1528,12 @@ static struct i2c_board_info lis331dlh_i2c_boardinfo[] = {
 		I2C_BOARD_INFO("lis331dlh", 0x18),
 		.platform_data = &lis331dlh_pdata,
 	},
+	{
+		I2C_BOARD_INFO("tsl2550", 0x39),
+	},
+	{
+		I2C_BOARD_INFO("tmp275", 0x48),
+	},
 };
 
 static void lis331dlh_init(int evm_id, int profile)
-- 
1.7.5.4

